client/app.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Calendar from './components/Calendar.jsx';
+import rightArrow from './styles/icons/right_arrow.svg';
+import downArrow from './styles/icons/down_arrow.svg';
+import flag from './styles/icons/flag.svg';
+import question from './styles/icons/question.svg';
+import './styles/style.css';
+import $ from 'jquery';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.today = new Date();
+    this.state = {
+      year: this.today.getFullYear(),
+      month: this.today.getMonth(),
+      requestedDates: [],
+      stage: 0,
+      price: 0,
+      cleaning: 0,
+      maxGuests: 0,
+      minStay: 0,
+      serviceFee: 0,
+      bookedDates: []
```

Comment:
Suggest moving bookedDates up to be adjacent to requestedDates, as they are conceptually very closely related.